### ISSUE TITLE
Connecting to server via commandline starts a local game instead.

### ISSUE THREAD

**Issue by BluebirdGreycoat** [minetest/minetest#6083]
Which basically means that the script I was using to connect (since my admin password is rather long and bothersome to type) no longer works.

```
# Connect as Admin!
cd "${root}"
minetest --address "${address}" --port "${port}" --name "${name}" --password "$(cat ${passfile})" --go
```

Starting the client normally and putting address, port, name & password in manually works as expected.

**Comment by orwell96** [comment]
Are you sure that address is set and not an empty string?
The empty string for address is internally handled as "start a local game"
Maybe try the command line without variables

**Comment by BluebirdGreycoat** [comment]
Absolutely sure:
```
address="0.0.0.0"
```
located directly above the minetest command.
My script used to work flawlessly; it is something in Minetest that changed. I may have to try to find out what myself eventually.

**Comment by BluebirdGreycoat** [comment]
Actually it is. Specifically, it is a reserved IP with a special meaning, similar in class to `127.0.0.1`. Try pinging it.

I guess I forgot to mention that at the time I posted this issue, I was running the server on my local machine. That is no longer the case. This problem is also no longer particularly pressing for me. Despite my password's length, after 1 year I am now quite proficient in entering it. (I know I can change it ... what can I say, I'm lazy.)

I've recompiled the latest stable v0.4, it seems the problem is no longer present. I guess it got fixed and I didn't find out until now. This can probably be closed.
